runtime/testruntime/testdata/ad_bids/apis/simple_mv_sql_api_global.yaml

```diff
@@ -0,0 +1,4 @@
+kind : api
+
+metrics_sql: select dom from ad_bids_metrics LIMIT 10
+additional_where: dom != 'google.com'
```

Comment:
We don't use the `testruntime/testdata` projects any more (coupled tests got annoying past a certain amount of tests), instead create a project directly in your test cases with `testruntime.NewInstanceWithOptions` by providing files in the options.